plugins/quickinspector/quickinspector.cpp

```diff
@@ -228,6 +229,17 @@ static QString qsgTextureWrapModeToString(QSGTexture::WrapMode wrapMode)
   return QStringLiteral("Unknown: %1").arg(wrapMode);
 }
 
+static bool qGoodCandidateItem(QQuickItem *item)
```

Comment:
the "q" prefix indicates to the incautious reader, that Qt provides this function. is this intentional?